app/Console/Commands/Debug.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Artisan;
+use App\Traits\Commands\EnvironmentWriterTrait;
+
+class Debug extends Command
```

Comment:
I don't think it makes sense to have the debug mode command. You are replacing one cli command `php artisan down/up` and a single .env line change with one cli command with two options and a choice input.